### PR TITLE
Ui accessibility labels added

### PR DESCRIPTION
**Pull request by bmourat**
Added accessibility labels for ui elements. One problem is that we can not set both hint and contentDescription attributes for EditText. I decided to display info if a field is mandatory in a hint, since screen readers read hints anyway. We also need to provide translations for this hints.

### CODE REVIEW

**Comment by bmourat** [comment]
Hints messages added to all language files to pass Travis build

File: hockeysdk/src/main/java/net/hockeyapp/android/FeedbackActivity.java [hunk 1]
```diff
@@ -56,9 +56,8 @@
 
 /**
  * <h3>Description</h3>
- *
+ * <p>
```
Comment (TroubleMakerBen) [inline hockeysdk/src/main/java/net/hockeyapp/android/FeedbackActivity.java]: Please remove the p-tag here.

**Comment by TroubleMakerBen** [comment]
Please use the following translations...in general we want "Fieldname (required)", e.g. `Password (Required)`.


```
EN: $FieldName (Required)
ZH: $FieldName (必填)
RU: $FieldName (Обязательное поле)
PT: $FieldName (Necessário)
IT: $FieldName (Richiesto)
FR: $FieldName (Requis)
ES: $FieldName (Obligatorio)
DE: $FieldName (Erforderlich)
JP: $FieldName (必填)
HR: $FieldName (Obavezno)
HU: $FieldName (Szükséges mező)
NL: $FieldName (Verplicht)
FA: (مورد نیاز) $FieldName 
NB: $FieldName (Påkrevd)
```

The value for "FieldName" can be found in other parts of the localization files.

**Comment by bmourat** [comment]
There is strange issue with preview on github. When i open values-fa strings in android studio everything looks like it should be, and on device with persian language it also looks fine, but on github order of letters is somehow mixed up.

**Pull request merged**